ga4gh/datamodel/datasets.py

```diff
@@ -227,6 +226,35 @@ def __init__(self, localId, dataDir, dataRepository):
                     self, localId, bamPath, dataRepository)
                 self.addReadGroupSet(readGroupSet)
 
+    def _metadataFileExists(self, fname):
```

Comment:
These should be moved into the VariantSet class. There are probably existing methods that can be used also, since a few others use this JSON metadata framework.